yb-voyager/cmd/common.go

```diff
@@ -87,7 +87,9 @@ func getMappingForTableNameVsTableFileName(dataDirPath string) map[string]string
 		time.Sleep(time.Second * 1)
 	}
 
-	pgRestoreCmd := exec.Command("pg_restore", "-l", dataDirPath)
+	var pgRestorePath string
+	srcdb.FindCorrectPGCommandPath(&pgRestorePath, "pg_restore")
```

Comment:
Why not just return the path from the function e.g.

    pgRestorePath := srcdb.FindCorrectPGCommandPath("pg_restore")